### PR TITLE
Remove useless files in model compression examples

### PR DESCRIPTION
**Pull request by colorjam**

### CODE REVIEW

**Comment by QuanluZhang** [comment]
We cannot directly remove the code for reproducing pruner's results. suggest to create two directories: pruning and quantization. In pruning, there could be a directory called reproduced.

File: docs/zh_CN/CommunitySharings/ModelCompressionComparison.md [hunk 1]
```diff
@@ -30,15 +30,15 @@
...
 
 CIFAR-10, ResNet18:
 
-![](../../../examples/model_compress/comparison_of_pruners/img/performance_comparison_resnet18.png)
+![](../../../examples/model_compress/pruning/comparison_of_pruners/img/performance_comparison_resnet18.png)
 
 CIFAR-10, ResNet50:
 
-![](../../../examples/model_compress/comparison_of_pruners/img/performance_comparison_resnet50.png)
+![](../../../examples/model_compress/pruning/comparison_of_pruners/img/performance_comparison_resnet50.png)
```
Comment (QuanluZhang) [inline docs/zh_CN/CommunitySharings/ModelCompressionComparison.md:41]: do not modify chinese doc

Comment (colorjam) [reply docs/zh_CN/CommunitySharings/ModelCompressionComparison.md]: Thanks, fix it, please check the latest version~

**Review by QuanluZhang — APPROVED** [review]

File: examples/model_compress/pruning/reproduced/L1_torch_pruner.py [hunk 2]
```diff
@@ -118,4 +118,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (J-shang) [inline examples/model_compress/pruning/reproduced/L1_torch_pruner.py:121]: better to add a newline at the end

File: examples/model_compress/pruning/reproduced/slim_torch_pruner.py [hunk 3]
```diff
@@ -142,4 +142,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (J-shang) [inline examples/model_compress/pruning/reproduced/slim_torch_pruner.py:145]: the same need newline, but unnecessary

**Review by J-shang — APPROVED** [review]

File: examples/model_compress/pruning/reproduced/slim_torch_pruner.py (hunk 3 above)
Comment (colorjam) [reply examples/model_compress/pruning/reproduced/slim_torch_pruner.py]: Added, thanks~

**Pull request merged**